octoprint_octorelay/driver.py

```diff
@@ -11,7 +11,7 @@ class Driver():
     def __init__(self, pin: int, inverted: bool, pin_factory=None):
         self.pin = pin # GPIO pin
         self.inverted = inverted # marks the relay as normally closed
-        self.handle = LED(pin, pin_factory=pin_factory)
+        self.handle = LED(pin, pin_factory=pin_factory, initial_value=None)
```

Comment:
or maybe it should be `not inverted`...

Reply:
tested locally:
- `None` works good for inverted relays
- `inverted` works good for all relays

```suggestion
        self.handle = LED(pin, pin_factory=pin_factory, initial_value=inverted)
```